WebHostLib/misc.py

```diff
@@ -37,23 +40,26 @@ def start_playing():
     return render_template(f"startPlaying.html")
 
 
-# TODO for back compat. remove around 0.4.5
-@app.route("/weighted-settings")
-def weighted_settings():
-    return redirect("weighted-options", 301)
-
-
-@app.route("/weighted-options")
-@cache.cached()
-def weighted_options():
-    return render_template("weighted-options.html")
-
-
-# Player options pages
-@app.route("/games/<string:game>/player-options")
+@app.route("/games/<string:game>/option-presets", methods=["GET"])
 @cache.cached()
-def player_options(game: str):
-    return render_template("player-options.html", game=game, theme=get_world_theme(game))
+def option_presets(game):
```

Comment:
```suggestion
def option_presets(game: str) -> Response:
```

Reply:
Done, and I moved the route into `options.py`, to better categorize it with other relevant routes.